cleanup.yml

```diff
@@ -4,9 +4,9 @@
   collections:
   - ibm.cloudcollection
   tasks:
-    - name: Fetch the variables from var file
-      include_vars:
-        file: vars.yml
+    - name: Load appropriate vars
+      include_role:
+        name: load_vars
```

Comment:
Loading the vars here in the same way as the create.yml ensures everything that was created gets destroyed (some logic in here makes sure # of execution nodes etc is set the same)